src/passkey_child/passkey_child_assert.c

```diff
@@ -125,6 +156,69 @@ set_assert_options(fido_opt_t up, fido_opt_t uv, fido_assert_t *_assert)
     return ret;
 }
 
+errno_t
+get_assert_auth_data_signature(TALLOC_CTX *mem_ctx, fido_assert_t *assert,
+                               const char **_auth_data,
+                               const char **_signature)
+{
```

Comment:
Lots of functions here returns a pointer but it is never freed on error.

Reply:
As an example are you referring to `fido_assert_authdata_ptr()`?

Reply:
Ah, I see these are just returning ptr from the assert, but it is not duplicating it.